visualizer_drag_gradio.py

```diff
@@ -13,10 +13,15 @@
                           get_latest_points_pair, get_valid_mask,
                           on_change_single_global_state)
 from viz.renderer import Renderer, add_watermark_np
+import inspect
 
 parser = ArgumentParser()
 parser.add_argument('--share', action='store_true',default='True')
 parser.add_argument('--cache-dir', type=str, default='./checkpoints')
+parser.add_argument('--listen', action='store_true',
+                    help="launch gradio with 0.0.0.0 as server name, allowing to respond to network requests")
```

Comment:
How about changing this argument to `--host`, so that users can set the host IP as they prefer?

Reply:
I need to hack WhatsApp